etc/base.yaml.gotmpl

```diff
@@ -27,8 +27,9 @@ radar_grafana:
 #  google_application_credentials: {{ readFile "../etc/radar-appserver/firebase-adminsdk.json" | quote }}
 #*/}}
 
-
-ksql_server:
-  ksql:
-    queries: |
-      {{- readFile "cp-ksql-server/queries.sql" | nindent 8 }}
\ No newline at end of file
+# Remove below Go comment to read the queries.sql and set the queries
+# in the ksql_server
+#ksql_server:
+#  ksql:
+#    queries: |
+#      {{- readFile "cp-ksql-server/queries.sql" | nindent 8 }}
```

Comment:
You also need to comment within the quotes as well (same as others above)
```
{{/*
....
*/}}
```

Otherwise, it causes issues with go templating.

Reply:
oh alright, will do that.